src/components/Main.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import CatPics from "./catPics";
+function Main() {
```

Comment:
Good work so far!

We can indent one space between our imports and our class components for easier readability 👍